video3.html

```diff
@@ -1,4 +1,4 @@
-  <a href='/'><button>Voltar</button></a>
+  <a href='/cursos/curso1'><button>Voltar</button></a>
   <p>Node.js API testável #3 - Routes e Database com Sequelize e Sqlite3</p>
   <iframe width="560" height="315" src="https://www.youtube.com/embed/ixvwD1ZMGLk" frameborder="0" allowfullscreen></iframe>
     <br><br><a href='/video2'><button>Anterior</button></a>
```

Comment:
A tag "<br">" não deve ser utilizada para separar elementos.

Mais sobre isso: https://www.tipsandtricks-hq.com/10-common-html-mistakes-to-avoid-1980